src/components/CopyUrlButton/CopyUrlButton.visual.tsx

```diff
@@ -0,0 +1,34 @@
+import * as React from 'react';
+import { storiesOf } from '@storybook/react';
+import { VisualTestContainer } from '../../../test/visual/VisualTestContainer';
+import { CopyUrlButton } from './';
+
+class CopyUrlButtonVisual extends React.Component<any> {
+  render() {
+    return (
+      <VisualTestContainer>
+        <CopyUrlButton {...this.props} />
+      </VisualTestContainer>
+    );
+  }
+}
+
+const tests = [
+  {
+    describe: 'basic',
+    its: [
+      {
+        it: 'default',
+        props: {},
+      },
+    ],
+  },
+];
+
+tests.forEach(({ describe, its }) => {
+  its.forEach(({ it, props }) => {
+    storiesOf(`CopyUrlButton/${describe}`, module).add(it, () => (
+      <CopyUrlButtonVisual {...props} />
+    ));
+  });
+});
```

Comment:
can you use storybook-snapper instead?
You can see an example here:
https://github.com/wix/wix-ui-tpa/blob/master/src/components/NewCard/NewCard.visual.tsx

It's my bad that I hadn't added this to the generator yet.

Reply:
done